CMakeLists.txt

```diff
@@ -1,121 +1,86 @@
 cmake_minimum_required(VERSION 3.0)
-project(ftxui
+project(
+  ftxui
   LANGUAGES CXX
-  VERSION 0.1
-)
+  VERSION 0.1)
 
 enable_testing()
 
 cmake_minimum_required(VERSION 3.0)
+
+option(FTXUI_BUILD_EXAMPLES "Set to ON to build examples" ON)
+option(FTXUI_BUILD_TESTS "Set to ON to build tests" ON)
+option(FTXUI_ENABLE_INSTALL "Generate the install target" ON)
```

Comment:
Could you explain why we need the "FTXUI_ENABLE_INSTALL" option?
Why would the user set this to OFF?

Reply:
If the user wants to use the library as a dependency for a single program and does not want to install it globally in the system, the ability to remove the install target slightly speeds up the build

Reply:
```
and does not want to install it globally in the system
```
I don't understand how the user who doesn't want to install FTXUi would end up install it?
You are probably right, but I don't understand.

```
the ability to remove the install target slightly speeds up the build
```
I am not convinced this can be humanly perceivable.